intro-HPC/ch_troubleshooting.tex

```diff
@@ -245,7 +245,7 @@ \section{Security warning about invalid host key}
 \end{prompt}
 
 You will need to remove the line it's complaining about (in the example, line 21).
-To do that, open \lstinline|~/.ssh/config| in an editor, and remove the line. This
+To do that, open \lstinline|~/.ssh/known_hosts| in an editor, and remove the line. This
 results in \lstinline|ssh| ``forgetting'' the system you are connecting to.
```

Comment:
Actually, a better suggestion is to run `ssh-keygen -R name.of.host`, maybe mention that too as an alternative?

Reply:
does that work with mac?

Reply:
yes

Reply:
fixed